src/main/java/com/wuest/from_the_depths/integration/SSHelper.java

```diff
@@ -0,0 +1,97 @@
+package com.wuest.from_the_depths.integration;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSyntaxException;
+import net.minecraft.util.JsonUtils;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.common.Loader;
+import org.apache.commons.lang3.tuple.Pair;
+import sereneseasons.api.season.ISeasonState;
+import sereneseasons.api.season.Season;
+import sereneseasons.api.season.SeasonHelper;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BiPredicate;
+import java.util.function.BooleanSupplier;
+
+/**
+ * Helper class that contains Utility Methods to interact with Serene Seasons.
+ * (trying to keep the mod as an Optional Soft Dependency)
+ * @author Davoleo
+ */
+public class SSHelper {
```

Comment:
Please re-name this class. My mind went to a bad place when I saw "SS".

Reply:
oof yeah sure, sorry about this